serverlessworkflow/modules/ROOT/pages/getting-started/java-embedded-workflows.adoc

```diff
@@ -0,0 +1,106 @@
+= Workflow embedded execution in Java 
+
+This guide uses a standard Java virtual machine and a small set of Maven dependencies to execute a link:{spec_doc_url}[CNCF Serverless Workflow] definition. Therefore, it is assumed you are fluent both in Java and Maven. 
+The workflow definition to be executed can be read from a `.json` or `.yaml` file or programmatically defined using the {product_name} fluent API. 
+
+[[embedded-file-quick-start]]
+== Hello world (using existing definition file)
+
+The first step is to set up an empty Maven project with the following dependency:
+
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-serverless-workflow-executor-core</artifactId>
+  <version>RELEASE</version>
+</dependency>
+----
+
+Also, you might optionally add a `simple logger for java` dependency to avoid using `System.out.println`
```

Comment:
```suggestion
This guide uses [slf4j dependency](https://mvnrepository.com/artifact/org.slf4j/slf4j-simple). Consider using any other logger for java to avoid using `System.out.println`
```